Cargo.toml

```diff
@@ -21,3 +21,7 @@ diesel = { version = "1.4.5", features = ["chrono", "postgres", "r2d2"] }
 rand = "0.7.3"
 rocket = "0.4.5"
 rocket_contrib = {version = "0.4.5", features = ["json"]}
+serde = { version = "1.0", features = ["derive"] }
+
+[dev-dependencies]
+serde_json = "1.0"
```

Comment:
Rocket depends on serde_json. We are not using it directly in the production code, only for unit tests.